static/js/settings_account.js

```diff
@@ -130,7 +128,7 @@ exports.append_custom_profile_fields = function (element_id, user_id) {
         const field_choices = [];
 
         if (field_value === undefined || field_value === null) {
-            field_value = {value: "", rendered_value: ""};
+            field_value = { value: "", rendered_value: "" };
```

Comment:
Can you remove these automated changes your editor seems to have added to this pull request?  It makes it very hard to read your changes when they are mixed with random style changes.

Reply:
Yes. Sure. Sorry.